zilliqa/src/cfg.rs

```diff
@@ -518,6 +519,8 @@ pub struct ForkDelta {
     /// the contract balance will be sum of the existing balance and the amount sent in the deployment transaction.
     /// If false, the contract balance will be the amount sent in the deployment transaction.
     pub scilla_contract_creation_increments_account_balance: Option<bool>,
+    // If true then the compiled code is using serde_json with feature "preserve_order"
+    pub serde_json_preserve_ordering: Option<bool>,
```

Comment:
nit: I feel the feature should be named something like `init_data_preserve_order` instead. The user/configuration shouldn't need to know the implementation detail that we happen to use `serde_json` internally.